### ISSUE TITLE
Import/Reimport of a Test with 0 Findings fails with error when using scan type IntSights Report

### ISSUE THREAD

**Issue by 37b** [DefectDojo/django-DefectDojo#6274]
**Bug description**
Utilizing the "close old Findings" pattern to close Findings not found in the most recent import, Defect Dojo will fail to close the last Finding if the most recent import has no Findings at all. Instead, the application throws an error.

**Steps to reproduce**
1. Create Test with Type = IntSights Report with one or more Findings. 
2. Use the Re-Upload feature from the UI (or `reimport-scan` API), upload a new version of the test results with no active Findings.

**Expected behavior**
The one or more Findings should be closed as the latest report shows no open Findings.

**Deployment method** *(select with an `X`)*
- [X] Docker Compose
- [ ] Kubernetes
- [ ] GoDojo

**Environment information**
 - Operating System: Amazon Linux 2
 - DefectDojo version (see footer) or commit message: 2.7.1

**Logs** 
```
[09/May/2022 19:50:33] ERROR [dojo.api_v2.exception_handler:32] No alert in the report
--
Traceback (most recent call last):
File "/app/./dojo/api_v2/serializers.py", line 1513, in save
reimporter.reimport_scan(scan, scan_type, test, active=active, verified=verified,
File "/app/./dojo/importers/reimporter/reimporter.py", line 306, in reimport_scan
parsed_findings = parser.get_findings(scan, test)
File "/app/./dojo/tools/intsights/parser.py", line 149, in get_findings
raise ValueError('No alert in the report')
ValueError: No alert in the report
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
response = handler(request, *args, **kwargs)
File "/usr/local/lib/python3.8/site-packages/rest_framework/mixins.py", line 19, in create
self.perform_create(serializer)
File "/app/./dojo/api_v2/views.py", line 2075, in perform_create
serializer.save(push_to_jira=push_to_jira)
File "/app/./dojo/api_v2/serializers.py", line 1565, in save
raise Exception(ve)
Exception: No alert in the report
[09/May/2022 19:50:33] ERROR [django.request:224] Internal Server Error: /api/v2/reimport-scan/
ERROR:django.request:Internal Server Error: /api/v2/reimport-scan/
```

**Comment by 37b** [comment]
Happy to take this one since I introduced it!

**Comment by StefanFl** [comment]
merged for release 2.11
